script/1_0_2current_en_script/Ciel/Day 3/03_01_CIEL03_2_1.txt

```diff
@@ -20,7 +20,7 @@ The bathrooms in the mansion are huge, Arihiko. They're bigger than your room. A
 Thank you, Arihiko, for your encouraging response.
 It's a relief to hear something that I can agree with wholeheartedly.
 <Page6>
-"...Well, it can't be helped if we're talking about the mansion itself, though.
+"...Well, it can't be helped if we're talking about the mansion itself.
 Even if I get up at seven in the morning, they complain that I'm late, and the curfew is at eight in the evening. On top of that, if I stay out overnight without permission, I'll get interrogated. It's like living in a prison, right?"
```

Comment:
Liberal TL modifying sentence structure because the し,し reads kinda awkwardly in english. mixing of o'clock and PM is intentional, I think it reads well but open to change it. The but in place of "and" is just trying to drive home the logical inconsistency
> 朝は７時に起きても遅いと文句は言われるし、門限は８時だし。おまけに無断外泊すると厳しく詰問されるなんて、監獄生活みたいだろ？」
They complain that I'm sleeping in even if I wake up at 7 o'clock in the morning, but curfew is at 8PM. To make matters worse, if I stay out overnight without permission, I'll get interrogated. It's like living in a prison, right?